javagen/src/main/java/com/azure/autorest/template/ServiceAsyncClientTemplate.java

```diff
@@ -144,6 +144,10 @@ protected void addGeneratedAnnotation(JavaContext classBlock) {
     classBlock.annotation(Annotation.GENERATED.getName());
   }
 
+  protected boolean isPartialUpdateSupported() {
+    return true;
```

Comment:
Any reasons this just always returns true while other cases are based on the Autorest setting for enabling partial update? To be clear, I like this better where it's always true but was just wondering.